app/helpers/admin/taggable_content_helper.rb

```diff
@@ -93,6 +93,14 @@ def taggable_world_locations_container
     end
   end
 
+  # Returns an Array that represents the taggable roles. Each element of the
```

Comment:
Absolutely not a complaint, but this file uses a lot of comments and we tend to not use them, any idea why?

Reply:
I've no idea as I just copied this from another example.  I'm tempted to remove all the comments completely, as I don't think the code is complicated enough to need an explanation.

Reply:
I'll do that as a separate PR.

Reply:
I guess if you do that it'll be one to ask publishing experience about just to make sure they're happy and there's not a bigger reason that they're here